testdata/foo.1234.js

```diff
@@ -0,0 +1,2 @@
+require=function r(n,o,e){function t(f,i){if(!o[f]){if(!n[f]){var c="function"==typeof require&&require;if(!i&&c)return c(f,!0);if(u)return u(f,!0);var a=new Error("Cannot find module '"+f+"'");throw a.code="MODULE_NOT_FOUND",a}var p=o[f]={exports:{}};n[f][0].call(p.exports,function(r){var o=n[f][1][r];return t(o?o:r)},p,p.exports,r,n,o,e)}return o[f].exports}for(var u="function"==typeof require&&require,f=0;f<e.length;f++)t(e[f]);return t}({1:[function(r,n){function o(r){return"bar"+r}n.exports=o},{}],2:[function(r,n){var o=r("./bar"),e=function(r){return o(r)+o(r)};n.exports=e},{"./bar":1}],foo:[function(r,n){n.exports=r("./foo")},{"./foo":2}]},{},[]);
+//# sourceMappingURL=foo.min.js.map
```

Comment:
Since this points at `foo.min.js.map`, why would SME want to load the source map from `foo.1234.js.map`?

Reply:
True, copied from foo.min.js and didn’t check the contents.

Reply:
```suggestion
//# sourceMappingURL=foo.1234.js.map
```